shardy/dialect/sdy/ir/attrs.td

```diff
@@ -74,8 +75,8 @@ def Sdy_Mesh : AttrDef<Sdy_Dialect, "Mesh"> {
       ordering: <["a"=3, "b"=2], device_ids=[0, 2, 4, 1, 3, 5]>
   }];
   let parameters = (ins
-      OptionalArrayRefParameter<"MeshAxisAttr">:$axes,
-      OptionalArrayRefParameter<"int64_t">: $device_ids
+      OptionalArrayRefParameter<"MeshAxisAttr", "List of mesh axes">:$axes,
```

Comment:
shall we also try to be consistent with using or not using "list of"? which do you think is better?

I would say Sdy_AxisRefList and Sdy_ListOfAxisRefLists should definitely have "list of" in the one field because that's what they are defining.

Reply:
Sounds good, I removed the "list of" from this description.